lib/adyen/rest.rb

```diff
@@ -29,7 +29,8 @@ def self.client
       Adyen::REST::Client.new(
         Adyen.configuration.environment,
         Adyen.configuration.api_username,
-        Adyen.configuration.api_password
+        Adyen.configuration.api_password,
+        Adyen.configuration.default_api_params
```

Comment:
I rather not use `Adyen.configuration.default_api_params`. Currently this is only used for the deprecated SOAP client; the REST client never uses it AFAIK. Is there any reason why you want this to be included?